ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -75,7 +75,16 @@ The following ansible plugins have been removed from the `arista.avd` Ansible co
 
 TODO: Level 3 sections for each change with details on how to migrate
 
-### AvdInterfaceDescriptions breaking changes
+### Base class for custom IP addressing `AvdIpAddressing` was moved
+
+The `AvdIpAddressing` class was moved from the Ansible collection to `pyavd.api.ip_addressing`. Import statements must be updated.
+
+```diff
+- from ansible_collections.arista.avd.roles.eos_designs.python_modules.ip_addressing import AvdIpAddressing
++ from pyavd.api.ip_addressing import AvdIpAddressing
+```
+
+### Base class for custom interface descriptions `AvdInterfaceDescriptions` breaking changes
```

Comment:
```suggestion
### Base class for custom interface descriptions `AvdInterfaceDescriptions` was moved (breaking changes)
```

Reply:
This one also contains changes to the signatures, so I kept the broader title. I am okay with changing it, but we need something that covers both.

Reply:
ohhh, its okay to keep as is then